modules/installation-creating-aws-subnet-localzone.adoc

```diff
@@ -0,0 +1,121 @@
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You opted in your AWS account to the Local Zone that hosts your VPC.
+* You generated the Ignition config files for your cluster.
+
+.Procedure
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "VpcId", <3>
+    "ParameterValue": "vpc-<random_string>" <4>
+  },
+  {
+    "ParameterKey": "PublicRouteTableId", <5>
+    "ParameterValue": "<vpc_rtb_pub>" <6>
+  },
+  {
+    "ParameterKey": "LocalZoneName", <7>
+    "ParameterValue": "<cluster_region_name>-<location_identifier>-<zone_identifier>" <8>
+  },
+  {
+    "ParameterKey": "LocalZoneNameShort", <9>
+    "ParameterValue": "<lz_zone_shortname>" <10>
+  },
+  {
+    "ParameterKey": "PublicSubnetCidr", <11>
+    "ParameterValue": "10.0.128.0/20" <12>
+  }
+]
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the `install-config.yaml` file for the cluster.
+<3> The VPC created resources will belong to.
+<4> Specify the `VpcId` value from the output of the CloudFormation template
+for the VPC.
+<5> The Public Route Table ID for the VPC.
+<6> Specify the `PublicRouteTableId` value from the output of the CloudFormation template for the VPC.
+<7>  The Local Zone name that the VPC belongs to.
+<8> Specify the Local Zone that you opted your AWS account into, such as `us-east-1-nyc-1a`.
+<9> The shortname of the Local Zone that the VPC belongs to.
+<10> Specify the Local Zone that you opted your AWS account into, such as `nyc1`.
+//How do we determine this shortname?
+<11> The CIDR block to allow access to the Local Zone.
+<12> Specify a CIDR block in the format `x.x.x.x/16-24`.
+//How do we know what this CIDR is?
+
+
+. Copy the template from the *CloudFormation template for the VPC*
+section of this topic and save it as a YAML file on your computer. This template
+describes the VPC that your cluster requires.
+
+. Launch the CloudFormation template to create a stack of AWS resources that represent the VPC:
++
+[IMPORTANT]
+====
+You must enter the command on a single line.
+====
++
+[source,terminal]
+----
+$ aws cloudformation create-stack --stack-name <name> <1>
+     --template-body file://<template>.yaml <2>
+     --parameters file://<parameters>.json <3>
```

Comment:
The previous note indicates that these must all be entered on a single line. Breaking a multi-line command like this has made users open bugs before because of rendering issues, IIRC.